apps/passport-server/src/apis/pretix/genericPretixAPI.ts

```diff
@@ -579,9 +595,11 @@ const GenericPretixEventSettingsSchema = z.object({
   attendee_emails_required: z.boolean()
 });
 
-const GenericPretixCategorySchema = z.object({
+// @rob - what are product categories?
```

Comment:
All tickets must be configured as "personalized", with the exception of "add-ons", and an add-on is recognized by its category. The use-case for non-personalized add-on tickets was Devconnect towels.

See https://github.com/proofcarryingdata/zupass/pull/1119 for original implementation.